src/containers/Permissions.spec.jsx

```diff
@@ -76,23 +85,7 @@ describe('Permissions', () => {
       fetchStatus: 'loaded',
       data: [
         {
-          attributes: {
-            permissions: {
-              files: {
-                type: 'io.cozy.files',
-                description: 'Required to access the files'
-              },
-              allFiles: {
-                type: 'io.cozy.files.*',
-                description: 'Required to access the files'
-              },
-              konnectors: {
-                description: 'Required to get the list of konnectors',
-                type: 'io.cozy.konnectors',
-                verbs: ['GET']
-              }
-            }
-          },
+          permissions: {},
```

Comment:
is it possible, a konnector with no permission? 🤔

Reply:
It'll do nothing, but there is nothing preventing it.